.changeset/config.json

```diff
@@ -1,12 +1,9 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@1.6.3/schema.json",
-  "changelog": [
-    "@changesets/changelog-github",
-    { "repo": "Developer-DAO/web3-ui" }
-  ],
+  "changelog": ["@changesets/changelog-github", { "repo": "Developer-DAO/web3-ui" }],
```

Comment:
This should not be changed. Can you make sure this change goes away please?

Reply:
sir, I almost lost my mind on this but I got the feeling this file was formatted by the commit hook.  I have pushed it multiple times just to notice that this file still has unwanted changes.